File: java/UKL/soal7_0.java
/*
Soal        : 7
Modelsoal   : 0
*/

import java.util.Scanner;

public class soal7_0 {
    static int hitung(int nilai) {
        int bayar=2000;
        if (nilai>10){
            bayar=bayar+1000;
        }
        if (nilai>20){
            bayar=bayar+1000;
        }
        if (nilai>30){
            bayar=bayar+1000;
        }

        return nilai*bayar+10000;
    }
 
    public static void main(String[] args) {
        Scanner inputt = new Scanner(System.in);
        int id, tagihan;
        String namaPelanggan[]={"Ali", "Budi", "Dani", "Edi", "Umar"};//nama pelanggan 
        
        System.out.println("Masukkan id pelanggan : 1");
        id = inputt.nextInt()-1;
        //id = 1-1;

        System.out.println("Masukkan jumlah tagihan : 17");
        tagihan = inputt.nextInt();
        //tagihan = 33;

        tagihan=hitung(tagihan);

        System.out.println(String.format(
            """
            Print Out Tagihan
            ID\t: %s
            Nama\t: %s
            Tagihan\t: Rp.%s,-
            """, 
            id+1, 
            namaPelanggan[id],
            tagihan));
    }
}

File: golang/UKL/soal7_0.go
/*
Soal        : 7
Modelsoal   : 0
*/

//namaPelanggan := [5]string{"Ali", "Budi", "Dani", "Edi", "Umar"} //nama pelanggan

package main

import "fmt"

func hitung(nilai int) int {
	bayar := 2000
	if nilai > 10 {
		bayar = bayar + 1000
	}
	if nilai > 20 {
		bayar = bayar + 1000
	}
	if nilai > 30 {
		bayar = bayar + 1000
	}

	return nilai*bayar + 10000
}

func input_int(msg string) int {
	fmt.Print(msg)
	var out int
	fmt.Scanln(&out)
	return out
}

func main() {
	namaPelanggan := [5]string{"Ali", "Budi", "Dani", "Edi", "Umar"}
	id := input_int("Masukkan jumlah pelanggan : ") - 1
	//println("Masukkan id pelanggan : 1");
	//id = 1-1;

	tagihan := input_int("Masukkan jumlah tagihan : ")
	//println("Masukkan jumlah tagihan : 33");
	//tagihan = 33;

	tagihan = hitung(tagihan)
	fmt.Println(id, namaPelanggan[0])
	//format print
}
